### DISCUSSION TITLE
Locksmith Possible Improvements

### DISCUSSION THREAD

**Discussion by Fl4sHNoX** [Malcolm-Q/LC-LateGameUpgrades#322] [Ideas]
Have a prompt appear on locked doors when the upgrade is purchased. 

Or have the locksmith feature be converted/made into some kind of device or item that you pickup (since I think it would probably be the "easiest" solution, I mean, in a way like keys that enable a prompt to unlock doors if that makes sense?).

**Comment by WhiteSpike** [comment]
Closing due to 3.3.0 implementating interaction with locked doors to trigger the locksmith minigame.
